resources/user-guides/attach-data.md

```diff
@@ -358,7 +363,7 @@ If you do not have existing rules for this property, the **Create New Rule** but
 
 On the right pane, you can add a freeform JSON payload that adds data to an SDK event before an extension that is listening for this event can hear the event. In this example, custom mbox and profile parameters are added to the event before the Target extension processes it. The added mbox and profile parameters will now be added on outgoing Target location displayed requests.
 
-In the following example, **extraKey**, **extraKey2**, and a data element defined for the **OS version** are added to the profile parameters of the Target event. A key named **customMboxParameter** is added to the mbox paramters of the Target event. Values for the new keys can either be hardcoded in the rule, or dynamically determined by the SDK when this event processes by using Data Elements.
+In the following example, **extraKey** and **extraKey2** are added to the profile parameters. A key named **customMboxParameter** and a data element defined for the **OS version** are added to the mbox parameters of the Target event. Values for the new keys can either be hardcoded in the rule, or dynamically determined by the SDK when this event processes by using Data Elements.
```

Comment:
- In the 2nd sentence, please add "that was" between "element" and "defined. 
- In the 3rd sentence, please add "be" between "or" and "dynamically" and remove the comma between "rule" and "or"
- In the 3rd sentence, please put "Data Elements" in lower case --> "data elements"